sky/cli.py

```diff
@@ -2916,7 +2925,7 @@ def show_gpus(
         all: bool,  # pylint: disable=redefined-builtin
         cloud: Optional[str],
         region: Optional[str]):
-    """Show supported GPU/TPU/accelerators.
+    """Show supported GPU/TPU/accelerators and their prices.
```

Comment:
May want to mention at L2935 that price is shown when GPU is specified or `--all` is used

Reply:
Done.